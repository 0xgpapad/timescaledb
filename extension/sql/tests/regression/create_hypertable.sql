\set ON_ERROR_STOP 1
\set VERBOSITY verbose
\set SHOW_CONTEXT never

\ir include/create_clustered_db.sql

\set ECHO ALL
\c meta
SELECT add_cluster_user('postgres', NULL);

SELECT set_meta('meta' :: NAME, 'localhost');
SELECT add_node('Test1' :: NAME, 'localhost');

SELECT add_node('test2' :: NAME, 'localhost');

\c Test1

create schema test_schema;
create table test_schema.test_table(time bigint, temp float8, device_id text);
\dt "test_schema".*
select * from create_hypertable('test_schema.test_table', 'time', 'device_id');

\C test2
\dt "test_schema".*
