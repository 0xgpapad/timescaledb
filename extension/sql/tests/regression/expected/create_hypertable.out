psql:include/create_clustered_db.sql:12: NOTICE:  00000: installing required extension "dblink"
LOCATION:  CreateExtensionInternal, extension.c:1441
psql:include/create_clustered_db.sql:12: NOTICE:  00000: installing required extension "postgres_fdw"
LOCATION:  CreateExtensionInternal, extension.c:1441
psql:include/create_clustered_db.sql:12: NOTICE:  00000: installing required extension "hstore"
LOCATION:  CreateExtensionInternal, extension.c:1441
 setup_meta 
------------
 
(1 row)

psql:include/create_clustered_db.sql:16: NOTICE:  00000: installing required extension "dblink"
LOCATION:  CreateExtensionInternal, extension.c:1441
psql:include/create_clustered_db.sql:16: NOTICE:  00000: installing required extension "postgres_fdw"
LOCATION:  CreateExtensionInternal, extension.c:1441
psql:include/create_clustered_db.sql:16: NOTICE:  00000: installing required extension "hstore"
LOCATION:  CreateExtensionInternal, extension.c:1441
 setup_main 
------------
 
(1 row)

psql:include/create_clustered_db.sql:20: NOTICE:  00000: installing required extension "dblink"
LOCATION:  CreateExtensionInternal, extension.c:1441
psql:include/create_clustered_db.sql:20: NOTICE:  00000: installing required extension "postgres_fdw"
LOCATION:  CreateExtensionInternal, extension.c:1441
psql:include/create_clustered_db.sql:20: NOTICE:  00000: installing required extension "hstore"
LOCATION:  CreateExtensionInternal, extension.c:1441
 setup_main 
------------
 
(1 row)

\c meta
SELECT add_cluster_user('postgres', NULL);
 add_cluster_user 
------------------
 
(1 row)

SELECT set_meta('meta' :: NAME, 'localhost');
 set_meta 
----------
 
(1 row)

SELECT add_node('Test1' :: NAME, 'localhost');
 add_node 
----------
 
(1 row)

SELECT add_node('test2' :: NAME, 'localhost');
 add_node 
----------
 
(1 row)

\c Test1
create schema test_schema;
create table test_schema.test_table(time bigint, temp float8, device_id text);
\dt "test_schema".*
              List of relations
   Schema    |    Name    | Type  |  Owner   
-------------+------------+-------+----------
 test_schema | test_table | table | postgres
(1 row)

select * from create_hypertable('test_schema.test_table', 'time', 'device_id');
 create_hypertable 
-------------------
 
(1 row)

\C test2
\dt "test_schema".*
              List of relations
   Schema    |    Name    | Type  |  Owner   
-------------+------------+-------+----------
 test_schema | test_table | table | postgres
(1 row)

